src/components/donation/Steps/Donater/Form/index.tsx

```diff
@@ -15,38 +17,64 @@ export default function Form(props: {
 }) {
   const {
     reset,
+    resetField,
     handleSubmit,
     getValues,
+    watch,
     formState: { isValid, isDirty, isSubmitting },
-  } = useFormContext<DonateValues>();
+  } = useFormContext<DV>();
 
   const isInsideWidget = useIsInsideWidget();
 
   const endowId = useGetter((state) => state.donation.recipient?.id);
 
   const dispatch = useDispatch();
 
-  function submit(data: DonateValues) {
+  function submit(data: DV) {
     dispatch(setDetails(data));
     reset();
   }
 
   const wasCompleted = !!getValues("token.amount");
 
+  const tokenType = watch("token.type");
+
   return (
     <form
       onSubmit={handleSubmit(submit)}
       className="grid rounded-md w-full"
       autoComplete="off"
     >
-      <TokenField<DonateValues, "token">
+      <TokenField<DV, "token">
```

Comment:
nit: Better for our future selves and new devs to leave this spelled out than to shorten into the more cryptic `DV` abbreviation.